src/test/java/com/google/maps/DirectionsApiTest.java

```diff
@@ -26,89 +26,108 @@
 import com.google.maps.model.TransitRoutingPreference;
 import com.google.maps.model.TravelMode;
 import com.google.maps.model.Unit;
+import java.util.ArrayList;
```

Comment:
I'd agree, but [google-java-format](https://github.com/google/google-java-format) seems happy with it.